### ISSUE TITLE
"Script DAX" option on calc items opens a blank DAX Script tab

### ISSUE THREAD

**Issue by bernatagulloesbrina** [TabularEditor/TabularEditor3#816]
It would be nice if right-click on a calc item and selecting "DAX Script" created the DAX script for that calc item. Right now it opens an empty DAX script tab

**Comment by mlonsk** [comment]
Hi @bernatagulloesbrina 

Thank you for reporting, we will look into if it is possible to do. 

You probably already know, but you can always script the whole calculation group which will give you the script for all calculation items.

**Comment by bernatagulloesbrina** [comment]
Yes, that way it does work, but sometimes i just want the code of certain
Calc item.

Not a deal breaker, but would be nice if it worked
